frontend/app/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.compose.compiler)
+}
+
+android {
+    namespace = "de.amosproj3.ziofa"
+    compileSdk = 34
+
```

Comment:
could you please change `compileSdk = 35` and add `buildToolsVersion = "35.0.0"`

Currently for the rust part and the docker image we are getting the 35 versions and having that in sync would be nice, otherwise we need two versions of everything (platforms-35 and platforms-34, buildtools-34.0.0 and buildtools-35.0.0).

Otherwise I could also bump the versions for our docker image down, however if the higher versions work it would be nice.

Reply:
done